tests/FieldTypesTest.php

```diff
@@ -31,21 +31,45 @@ public function testEmailBasic()
         $m = new Model($this->pers);
         $m->addField('email', [Field\Email::class]);
 
+        // null value
+        $m->set('email', null);
+        $m->save();
+        $this->assertNull($m->get('email'));
+
+        // normal value
         $m->set('email', 'foo@example.com');
         $m->save();
         $this->assertSame('foo@example.com', $m->get('email'));
 
-        // padding removed
+        // padding, spaceing etc removed
         $m->set('email', " \t " . 'foo@example.com ' . " \n ");
         $m->save();
         $this->assertSame('foo@example.com', $m->get('email'));
 
+        // no domain - go to hell :)
         $this->expectException(ValidationException::class);
         $this->expectExceptionMessage('does not have domain');
         $m->set('email', 'qq');
     }
 
-    public function testEmailMultiple()
+    public function testMultipleEmailFields()
```

Comment:
I do not think we need this test

Reply:
I added this test here to show how it's working (see https://github.com/atk4/data/issues/450).
But probably you're right that this do not solve anything, just adds another test to prove that #450 can be closed :)

Reply:
addFields is absolutely unrelated with email, I do not think we should test it, it also add LOC to maintain